Podfile.lock

```diff
@@ -1,5 +1,5 @@
 PODS:
-  - Alamofire (4.8.0)
+  - Alamofire (4.8.2)
```

Comment:
I wasn't expecting this to change version after it being removed from the `Podfile` and `podspec` 🤔

Reply:
Ah! Nevermind, it's a dependency at [the WordPressKit level](https://github.com/wordpress-mobile/WordPressKit-iOS/blob/79ebd209c439a3947b9eed788050a9f414ce87c0/WordPressKit.podspec#L25).